ansible/playbooks/jenkins/worker/fix.yml

```diff
@@ -0,0 +1,60 @@
+---
+
+#
+# Fix common problems with Jenkins workers
+#
+
+- hosts:
+  - test
+  - release
+  # Benchmarking machine
+  - infra-softlayer-ubuntu1404-x64-2
+
+  tasks:
+    # Clean disk space
+    - name: Clean disk space
+      block:
+        - name: Get disk space
+          command: "df -h"
+          register: disk_usage
+        - set_fact:
+            disk_usage: "{{ disk_usage.stdout_lines }}"
+        - debug:
+            var: "{{ disk_usage }}"
+
+        - name: Get directories to delete
+          find:
+            paths: /home/iojs/build/workspace
+            patterns: "node-test-binary-arm,node-test-commit-*,libuv-test-commit-*,citgm-*"
```

Comment:
Just delete everything inside the workspace. Perhaps we should make this explicit in the docs somewhere: there should be nothing of value anywhere in the workers, we should be able to delete and recreate workers at any time.